src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -188,6 +188,10 @@ var DOMPropertyOperations = {
     }
     if (value == null) {
       node.removeAttribute(name);
+    } else if (value === true) {
+      node.setAttribute(name, '');
+    } else if (value === false) {
+      node.removeAttribute(name);
```

Comment:
This change worries me. This would be breaking behavior since we currently just stringify `false` and this would potentially change runtime behavior. I think we should leave this specific part as-is, where `null` is used to identify attributes that should be removed.